utils/validationUsers.js

```diff
@@ -1,99 +1,131 @@
+import {
+  getItemLocalStorage,
+  setItemLocalStorage,
+  setItemSessiontorage,
+} from "../hooks/browserStorage.js";
 import { validationUserPassword } from "./validationUserPassword.js";
 
+const ERROR_MESSAGES = {
+  signin: ["이메일과 비밀번호를 확인해주세요."],
+  signup: [
+    "이미 사용 중인 아이디입니다.",
+    "비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.",
+    "비밀번호 확인이 올바르지 않습니다.",
+  ],
```

Comment:
```suggestion
  signup: {
    duplicatedId: "이미 사용 중인 아이디입니다.",
    ...: "비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.",
    ...: "비밀번호 확인이 올바르지 않습니다.",
  },
```
array의 index는 단순히 순서를 의미하기 때문에 object가 좋아보여요. 
나중에 typescript를 사용하시게 되면 enum을 사용하셔도 좋구요.

Reply:
반영하겠습니다!